docs/usage.md

```diff
@@ -26,7 +26,9 @@ from virtualizarr import open_virtual_dataset
 vds = open_virtual_dataset('air.nc')
 ```
 
-(Notice we did not have to explicitly indicate the file format, as {py:func}`open_virtual_dataset <virtualizarr.open_virtual_dataset>` will attempt to automatically infer it.)
+```{note}
+We did not have to explicitly indicate the file format because {py:func}`open_virtual_dataset <virtualizarr.open_virtual_dataset>` will attempt to automatically infer it.
+```
 
 Printing this "virtual dataset" shows that although it is an instance of `xarray.Dataset`, unlike a typical xarray dataset, in addition to a few in-memory numpy arrays, it also wraps {py:class}`ManifestArray <virtualizarr.manifests.ManifestArray>` objects.
```

Comment:
Let's link to the data structures page from here

Reply:
👍 link added in [`6ce00db` (#563)](https://github.com/zarr-developers/VirtualiZarr/pull/563/commits/6ce00db0758a19cdcc512d80d9d7542d540b634c)